Gems/AWSCore/Code/Source/Editor/UI/AWSCoreResourceMappingToolAction.cpp

```diff
@@ -11,124 +11,141 @@
-            if (!AZ::IO::SystemFile::Exists(m_toolReadMePath.c_str()))
-            {
-                AZ_Error("AWSCoreEditor", false, "Failed to find ResourceMappingTool README file at %s.", m_toolReadMePath.c_str());
-                m_toolReadMePath.clear();
-            }
+        m_toolReadMePath = AZStd::string::format("%s/%s/README.md", engineRootPath.c_str(), ResourceMappingToolDirectoryPath);
+        AzFramework::StringFunc::Path::Normalize(m_toolReadMePath);
+        if (!AZ::IO::SystemFile::Exists(m_toolReadMePath.c_str()))
+        {
+            AZ_Error(
+                AWSCoreResourceMappingToolActionName, false, "Failed to find ResourceMappingTool README file at %s.",
+                m_toolReadMePath.c_str());
+            m_toolReadMePath.clear();
+        }
 
-            char executablePath[AZ_MAX_PATH_LEN];
-            auto result = AZ::Utils::GetExecutablePath(executablePath, AZ_MAX_PATH_LEN);
-            if (result.m_pathStored != AZ::Utils::ExecutablePathResult::Success)
-            {
-                AZ_Error("AWSCoreEditor", false, "Failed to find engine executable path.");
-            }
-            else
+        char executablePath[AZ_MAX_PATH_LEN];
+        auto result = AZ::Utils::GetExecutablePath(executablePath, AZ_MAX_PATH_LEN);
+        if (result.m_pathStored != AZ::Utils::ExecutablePathResult::Success)
+        {
+            AZ_Error(AWSCoreResourceMappingToolActionName, false, "Failed to find engine executable path.");
+        }
+        else
+        {
+            if (result.m_pathIncludesFilename)
             {
-                if (result.m_pathIncludesFilename)
+                // Remove the file name if it exists, and keep the parent folder only
+                char* lastSeparatorAddress = strrchr(executablePath, AZ_CORRECT_FILESYSTEM_SEPARATOR);
+                if (lastSeparatorAddress)
                 {
-                    // Remove the file name if it exists, and keep the parent folder only
-                    char* lastSeparatorAddress = strrchr(executablePath, AZ_CORRECT_FILESYSTEM_SEPARATOR);
-                    if (lastSeparatorAddress)
-                    {
-                        *lastSeparatorAddress = '\0';
-                    }
+                    *lastSeparatorAddress = '\0';
                 }
             }
+        }
 
-            AZStd::string binDirectoryPath(executablePath);
-            auto lastSeparator = binDirectoryPath.find_last_of(AZ_CORRECT_FILESYSTEM_SEPARATOR);
-            if (lastSeparator != AZStd::string::npos)
-            {
-                m_isDebug = binDirectoryPath.substr(lastSeparator).contains("debug");
-            }
+        AZStd::string binDirectoryPath(executablePath);
+        auto lastSeparator = binDirectoryPath.find_last_of(AZ_CORRECT_FILESYSTEM_SEPARATOR);
+        if (lastSeparator != AZStd::string::npos)
+        {
+            m_isDebug = binDirectoryPath.substr(lastSeparator).contains("debug");
+        }
```

Comment:
This should use `AZ::IO::Path::Filename` to get the last path component

```suggestion
        m_isDebug = executablePath.Filename() == "debug";
```

That being said trying to detect whether an application is debug based on the path of where the executable is located is not a good approach.

The `AZ_BUILD_CONFIGURATION_TYPE` should be checked instead

```suggestion
        m_isDebug = AZStd::string_view(AZ_BUILD_CONFIGURATION_TYPE) == "debug";
```